tools/collect_metrics.py

```diff
@@ -220,36 +223,61 @@ def generate_md_for_input_variations(self) -> str:
         Returns:
             Markdown string
         """
-        # Create a high level table of each op first
+        md = self.generate_md_for_high_level_table()
+        md += "***\n"
+
+        for opname, input_vars in self.sorted().items():
+            input_vars_dict = input_vars.get_list_input_var_to_dict()
+            md += f"### {opname}\n"
+            md += pd.DataFrame(input_vars_dict).to_markdown(index=True) + "\n"
+
+        return md
+
+    def generate_md_for_high_level_table(self, links_to_ops=False):
+        # Create a high level table for the main page
         high_level_op_status = defaultdict(list)
-        sort_by_opname = dict(sorted(self.items()))
+        sort_by_opname = self.sorted()
         for opname, input_vars in sort_by_opname.items():
-            high_level_op_status["Operations"].append(opname)
+            ops = f"[{opname}](_ops/{opname}.md)" if links_to_ops else opname
+            high_level_op_status["Operations"].append(ops)
             high_level_op_status["Input Variations"].append(len(input_vars))
             high_level_op_status["Converted"].append(input_vars.get_conversion_status_count_for(ConversionStatus.DONE))
 
         md = ""
         md += f"# High Level Operations Status\n"
         md += pd.DataFrame(high_level_op_status).to_markdown(index=True) + "\n"
 
-        md += "***\n"
+        return md
 
-        for opname, input_vars in sort_by_opname.items():
-            input_vars_dict = input_vars.get_list_input_var_to_dict()
-            md += f"### {opname}\n"
-            md += pd.DataFrame(input_vars_dict).to_markdown(index=True) + "\n"
+    def write_md(self, md: str, basedir: Path, filename: Path):
+        basedir.mkdir(parents=True, exist_ok=True)
+        with open(basedir / filename, "w") as text_file:
+            print(md, file=text_file)
+        print(f"Data written to {basedir / filename}")
 
-        return md
+    def write_md_for_cumulative_report(self):
+        md = self.generate_md_for_high_level_table()
+
+        basedir = Path("docs")
+        self.write_md(md, basedir, Path("cumulative_input_variations.md"))
+
+        # Create a new page for each op
+        cumulative_ops_dir = Path("_ops")
+        cumulative_ops_dir.mkdir(parents=True, exist_ok=True)
+
+        for opname, input_vars in self.sorted().items():
+            op_md = ""
+            input_vars_dict = input_vars.get_list_input_var_to_dict()
+            op_md += f"### {opname}\n"
+            op_md += pd.DataFrame(input_vars_dict).to_markdown(index=True) + "\n"
+            self.write_md(op_md, basedir / Path("_ops"), Path(f"{opname}.md"))
```

Comment:
```suggestion
            self.write_md(op_md, basedir / Path("operations"), Path(f"{opname}.md"))
```

this is a second time this string appears, should be a constant